### DISCUSSION TITLE
EDK2 BaseTools, can't make

### DISCUSSION THREAD

**Discussion by NonTemplate** [tianocore/edk2#4261] [General]
<img width="530" alt="f639584907760f93305032bc8b6497a" src="https://user-images.githubusercontent.com/103838511/231073873-f17f3c1e-2a41-4aa4-bed7-edb9cd595112.png">
I want to make the edk2 BaseTools. But ,I meet an error which shows in the picture. How to deal this error? I use the Ubuntu 18.04 operator system.  Thank you!

**Comment by ghost** [comment]
Which version of Python are you using? I think that happens when you have an older version.
If you have multiple versions installed, you can set the PYTHON_COMMAND environment variable to specify which to use.
